test/components/scroll-spy.spec.coffee

```diff
@@ -0,0 +1,61 @@
+{React, Testing, expect, _, sinon} = require './helpers/component-testing'
+
+ScrollSpy = require '../../src/components/scroll-spy'
+
+
+NestedComponent = React.createClass
+  render: -> React.createElement('div')
+
+RootComponent = React.createClass
+  render: ->
+
+    React.createElement('div', {},
+      React.createElement(ScrollSpy, @props,
+        React.createElement(NestedComponent, onScreenElements:[])
+      )
+    )
```

Comment:
there are a few ones in the [`./test/components`](https://github.com/openstax/tutor-js/tree/master/test/components)

![screen shot 2016-03-02 at 4 52 10 pm](https://cloud.githubusercontent.com/assets/2483873/13478570/3e09b242-e097-11e5-9c79-b24dba70814d.png)